js/src/utils/logger.ts

```diff
@@ -61,8 +61,8 @@ const logger = winston.createLogger({
   format: logFormat,
   transports: [
     new winston.transports.Console({
-      handleExceptions: true,
-      handleRejections: true
+      handleExceptions: false,
+      handleRejections: false
```

Comment:
Add a failure scenario test to avoid this.

Reply:
Added check in specs file